src/NuGet.Core/NuGet.Commands/RestoreCommand/Diagnostics/UnexpectedDependencyMessages.cs

```diff
@@ -172,14 +172,14 @@ public static RestoreLogMessage GetMissingLowerBoundMessage(ResolvedDependencyKe
         /// <summary>
         /// Warn for project dependencies that do not include a lower bound on the version range.
         /// </summary>
-        public static IEnumerable<RestoreLogMessage> GetProjectDependenciesMissingLowerBounds(PackageSpec project)
+        public static IEnumerable<RestoreLogMessage> GetProjectDependenciesMissingVersion(PackageSpec project)
         {
             return project.GetAllPackageDependencies()
                    .Where(e => HasMissingLowerBound(e.LibraryRange.VersionRange))
```

Comment:
Just wondering, this seems to detect that the version is missing a lower bound rather than missing a version all together.  Is that intentional?  I kind of expected some new logic that differentiated the two but is this how we detect a missing version?